ide/versioning.util/src/org/netbeans/modules/turbo/Memory.java

```diff
@@ -53,7 +53,7 @@
     public static final Object NULL = new Object();
 
     /** Keep reference to last isPrepared result. */
-    public static final ThreadLocal prepared = new ThreadLocal();
+    public static final ThreadLocal<Entry> prepared = new ThreadLocal<>();
```

Comment:
The users of `prepared` in this class could now drop their cast (I think else you are introducing another warning).

Reply:
Good catch!! It's fixed now..